crypto/cmp/cmp_msg.c

```diff
@@ -234,8 +236,7 @@ OSSL_CRMF_MSG *OSSL_CMP_CTX_setup_CRM(OSSL_CMP_CTX *ctx, int for_KUR, int rid)
              * it could be NULL if centralized key creation was supported
              */
             || !OSSL_CRMF_CERTTEMPLATE_fill(OSSL_CRMF_MSG_get0_tmpl(crm), rkey,
-                                            subject, ctx->issuer,
-                                            NULL /* serial */))
+                                            subject, issuer, NULL /* serial */))
```

Comment:
indentation looks wrong here..

Reply:
You mean the extra indentation (+ 4 chars) before the `||`?
I deliberately use this option here to avoid the same indentation as for the subsequent `goto err;`
See also the - still unofficial - update to the coding style rules at #10725.
And the comment above it is aligned to that indentation.

`check-format.pl` accepts this, while it found a wrongly indented `goto err;` on line 437, which I've just fixed.